openforcefield/topology/topology.py

```diff
@@ -1754,6 +1763,41 @@ def to_openeye(self,
 
         return oe_mol
 
+    def get_bond_between(self, i, j):
+        """Returns the bond between two atoms
+
+        Parameters
+        ----------
+        i, j : int or TopologyAtom
+            Atoms or atom indices to check
+
+        Returns
+        -------
+        bond : TopologyBond
+            The bond between i and j.
+
+        """
+        if (type(i) is int) and (type(j) is int):
+            atomi = self.atom(i)
+            atomj = self.atom(j)
+        elif (type(i) is TopologyAtom) and (type(j) is TopologyAtom):
+            atomi = i
+            atomj = j
+        else:
+            raise Exception(
+                "Invalid input passed to get_bond_between(). Expected ints or TopologyAtoms, "
+                "got {} and {}".format(i, j))
+
+        for top_bond in atomi.topology_bonds:
+            for top_atom in top_bond.atoms:
+                if top_atom == atomi:
+                    continue
+                if top_atom == atomj:
+                    return top_bond
+        # If atomj wasn't found in any of atomi's bonds, then they aren't bonded.
+        raise NotBondedError('No bond between atom {} and {}'.format(atomi, atomj))
+
+
     def is_bonded(self, i, j):
```

Comment:
Could this simply call `get_bond_between()`?

Reply:
Yes. I have no idea why I didn't do that. Fixed in c4969c7.